vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -248,6 +262,10 @@ public void saveAccount(@Nullable VimeoAccount vimeoAccount, String email) {
         mConfiguration.saveAccount(vimeoAccount, email);
     }
 
+    public void setConfiguration(@NotNull Configuration configuration) {
```

Comment:
Sorry, a couple more things :)...
- Should calling `setConfiguration` also call `setup`?  If not I think you're in the same situation as review #1 where various things will be out-of-sync.
-If so, then maybe we should rename this to `updateConfiguration`?

Reply:
See comment above.